tests/test_plugin.py

```diff
@@ -31,3 +38,28 @@ def test_stop_plugin_configuration_on_conn_error(mocked_get):
     expect(mocked_config._reportportal_configured is False,
            'The value of the _reportportal_configured is not False.')
     assert_expectations()
+
+
+@patch('pytest_reportportal.RPLogger.handle')
+@pytest.mark.parametrize("log_level", ("info", "debug", "warning", "error"))
+def test_logger_handle_attachment(mock_handler, logger, log_level):
+    """Test logger call for different log levels with some text attachment"""
+    log_call = getattr(logger, log_level)
+    attachment = "Some {} attachment".format(log_level)
+    log_call("Some {} message".format(log_level), attachment=attachment)
+    expect(mock_handler.call_count == 1, "logger.handle called more than 1 time")
+    expect(getattr(mock_handler.call_args[0][0], "attachment") == attachment,
+           "record.attachment in args doesn't match real value")
+    assert_expectations()
+
+
+@patch('pytest_reportportal.RPLogger.handle')
+@pytest.mark.parametrize("log_level", ("info", "debug", "warning", "error"))
+def test_logger_handle_no_attachment(mock_handler, logger, log_level):
+    """Test logger call for different log levels without any attachment"""
```

Comment:
One-line docstring should end with ".". https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

`"""Test logger call for different log levels without any attachment."""`